tokens/src/commands.rs

```diff
@@ -297,7 +297,27 @@ fn build_messages(
     stake_extras: &mut StakeExtras,
     created_accounts: &mut u64,
 ) -> Result<(), Error> {
-    for allocation in allocations.iter() {
+    let allocation_chunks = allocations.chunks(100); // 100 is the GetProgramAccounts limit
```

Comment:
nit: could you use the `solana_rpc_client_api::request::MAX_MULTIPLE_ACCOUNTS` instead?
also nit: could you move this down inside the `if let Some(...)` block since it's not needed outside of that?

Reply:
done!